elasticsearch/tests/elasticsearch/elasticsearch.tests.bom

```
@@ -41,6 +41,7 @@ brooklyn.catalog:
             id: elasticsearch-cluster
             brooklyn.config:
               cluster.initial.size: 1
+              elasticsearch.bind.address: _site_
```

could you add (below) instead as it will fail fast
```
brooklyn.parameters:
- name: elasticsearch.bind.address
  type: string
  constraints:
  - required
```

---

Or let this default if not set

---

_site_ is an elastic magic word meaning “Any site-local addresses on the system, for example 192.168.0.1.”

---

Cool i withdraw my previous suggestion ... this is much better